Dockerfile

```diff
@@ -1,16 +1,13 @@
-FROM alpine:3.13.5@sha256:def822f9851ca422481ec6fee59a9966f12b351c62ccb9aca841526ffaa9f748
+FROM alpine:3.14.1@sha256:be9bdc0ef8e96dbc428dc189b31e2e3b05523d96d12ed627c37aa2936653258c
 
 USER root
 
 ENTRYPOINT ["tini", "--"]
 
 RUN ["apk", "--no-cache", "upgrade"]
-RUN ["apk", "--no-cache", "add", "tini", "dnsmasq", "bash", "curl", "openssl"]
-# naxsi and awscli are not available from the main Alpine repositories yet.
-RUN ["apk", "--no-cache", "--repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing", "add", \
-     "nginx-naxsi=1.16.1-r0", "nginx-naxsi-mod-http-naxsi=1.16.1-r0", "nginx-naxsi-mod-http-xslt-filter=1.16.1-r0"]
+RUN ["apk", "--no-cache", "add", "tini", "dnsmasq", "bash", "curl", "openssl", "python3", "py-pip", "nginx-mod-http-naxsi=1.20.1-r3", "nginx-mod-http-xslt-filter=1.20.1-r3"]
 
-RUN ["apk", "--no-cache", "--repository=http://dl-cdn.alpinelinux.org/alpine/v3.13/community", "add", "aws-cli=1.18.177-r0"]
+RUN ["pip", "install", "awscli"]
```

Comment:
Which version of Python is alpine 3.14 using? Can we pin to a specific version of `awscli`?

Reply:
In theory we shouldn't need to, the major version bump for aws cli is under a different name in pip (awscliv2), if we lock to a specific patch version we will need to keep bumping it, and I think because we install with pip it won't be picked up by a docker scan either if there are security problems.

If the aws cli broke then our whole container wouldn't work since it's getting the config from s3 using the aws cli, as far as i can see that's literally all the aws cli does in this container, (as an aside I wish it was part of the go code instead, this whole image could be 100mb+ smaller (in that region anyway)) so it's a pretty binary did or didn't work

Reply:
OK - I'm just wary of being bitten by unexpected major bumps (or the other extreme of deprecated versions no longer working) on unpinned Python dependencies before. Agree that pinning to a patch version probably isn't ideal: can we do something like `RUN ["pip" "install" "awscli>=x.x,<y.y"]

Reply:
Understandable, awscli is a weird one in this regard since the major version has another package name (they were clearly very worried (rightly) about breaking everyones installs :) ) but pip does support pessimistic version constraints so you could do
```
pip install awscli~=1.20
```

to get anything between 1.20 and 2.0 (exclusive of 2)

Reply:
Have added this constraint. It picks the correct version:
```
/ # pip install awscli~=1.20.0
Collecting awscli~=1.20.0
  Downloading awscli-1.20.27-py3-none-any.whl (3.7 MB)
```